### PR TITLE
added marker language support

### PR DESCRIPTION
**Pull request by a-guiducci**
### What's Changed?

added marker_langage attribute to the marker subclass of processed_text. 
updated testing.
apologies for the formatting changes 😮‍💨 

### PR Checklist

- [x] Updated unit tests
- [x] Ran unit tests

### CODE REVIEW

File: src/privateai_client/components/request_objects.py [hunk 1]
```diff
@@ -459,40 +437,56 @@ class ProcessedMarkerText(BaseRequestObject):
         "UNIQUE_NUMBERED_ENTITY_TYPE",
         "UNIQUE_HASHED_ENTITY_TYPE",
     ]
+    default_marker_language = "auto"
```
Comment (onurcayci) [inline src/privateai_client/components/request_objects.py:440]: The default is "en" under the docs. I think we should not turn this on by default like the web demo.

**Review by onurcayci — CHANGES_REQUESTED** [review]
Requested a change for the default marker language option.

File: src/privateai_client/components/request_objects.py (hunk 1 above)
Comment (a-guiducci) [reply src/privateai_client/components/request_objects.py:440]: ah shoot, you're right.. I don't know why I saw default as auto for some reason :/

**Review by onurcayci — APPROVED** [review]
LGTM! :+1: :100: :dollar: :heavy_dollar_sign:

**Pull request merged**